DH Jamy et al (#112)
ScilifelabDataCentre/data.scilifelab.se: content/highlights/diversity_of_life.md

---
title: New study shows crossing the salt barrier has shaped the diversity of life
date: 2022-10-19

> summary: TStudy by Jamy *et al.* show that hundreds of transitions have occurred between marine and non-marine habitats over the course of two billion years of eukaryotic evolution. Code and data are shared openly.

Reviewer: ```suggestion
summary: Study by Jamy *et al.* show that hundreds of transitions have occurred between marine and non-marine habitats over the course of two billion years of eukaryotic evolution. Code and data are shared openly.
```

banner: /highlights/diversity_of_life_thumbnail.png
banner_large: /highlights/diversity_of_life.png

> banner_caption: "Fig 1 from Jamy et al. (2022), Global eukaryotic 18S–28S phylogeny from environmental samples and the distribution of habitats."

Reviewer: ```suggestion
banner_caption: "Figure 1 from Jamy *et al.* (2022)."
```

> tags: [Highlight, Bayesian, Paleovegetation, Deep-learning]

Reviewer: these are not the right tags. We need to think of some. Ask Katarina. Highlight has to stay, try link to any related highlights with similar key words

> authors: [Katarina Öjefors Stark]

Reviewer: Katarina did not write this, the first author did, check with KÖS for full name

images: [/highlights/still_transition_model.png]
---

> Successfully colonizing a new habitat is a key evolutionary event. Over the course of animal and plant diversification, transitions between marine and non-marine habitats have seldom occurred. This habitat boundary, called the “salt-barrier”, is one of the hardest barriers to cross due to the strong shift in salinity and osmotic pressure. However, most eukaryotic diversity is in fact microbial—with larger population sizes and dispersal capabilities—and there is debate on how frequently these microbes have achieved marine/non-marine cross colonizations. The role of the salt barrier in shaping eukaryotic diversity at a broad scale is thus unknown.

Reviewer: ```suggestion
Successfully colonising a new habitat is a key evolutionary event. Over the course of animal and plant diversification, transitions between marine and non-marine habitats have seldom occurred. This habitat boundary, called the “salt-barrier”, is one of the hardest barriers to cross due to the strong shift in salinity and osmotic pressure. However, most eukaryotic diversity is in fact microbial (with larger population sizes and dispersal capabilities) and there is debate on how frequently these microbes have achieved marine/non-marine cross colonisations. The role of the salt barrier in shaping eukaryotic diversity at a broad scale is thus unknown.
```

> A recent study published in *Nature Ecology & Evolution* has now used a combination of long-read and short-read metabarcoding data to investigate how habitat preference has evolved across the eukaryotic tree of life. The work involved Uppsala University researchers along with a team of international collaborators (*First author:* Mahwash Jamy, Uppsala University; *Corresponding author:* [Fabien Burki,](https://www.scilifelab.se/researchers/fabien-burki/) Uppsala University/SciLifeLab).

Reviewer: Mahwash should also have a link to their profile.

> Jamy and colleagues first generated nearly 10 million long environmental reads (~4,500 bp spanning the ribosomal DNA operon) from 21 samples including Swedish freshwater lakes, permafrost thaw ponds, boreal and tropical forest soils, and oceans waters from the surface to the depths of the Mariana Trench. The long reads have increased phylogenetic signal, which (after clustering into ~17,000 taxonomically labeled OTUs) could be used to calculate phylogenetic trees spanning all major eukaryotic groups. Taxon sampling was further increased by incorporating publicly available short-read metabarcoding data (~234 million reads) from >1,500 samples globally (this dataset is available in a recently published article; see [Vaulot et al., 2022](https://doi.org/10.1111/1755-0998.13674) and [metaPR2 database](https://shiny.metapr2.org/metapr2/)).

Reviewer: ```suggestion
Jamy and colleagues first generated nearly 10 million long environmental reads (~4,500 bp spanning the ribosomal DNA operon) from 21 samples including Swedish freshwater lakes, permafrost thaw ponds, boreal and tropical forest soils, and oceans waters from the surface to the depths of the Mariana Trench. The long reads have increased phylogenetic signal, which (after clustering into ~17,000 taxonomically labeled OTUs) could be used to calculate phylogenetic trees spanning all major eukaryotic groups. Taxon sampling was further increased by incorporating publicly available short-read metabarcoding data (~234 million reads) from >1,500 samples globally (this dataset is available in a recently published article; see [Vaulot *et al.* (2022)](https://doi.org/10.1111/1755-0998.13674) and [metaPR2 database](https://shiny.metapr2.org/metapr2/)).
```

> These large, comprehensive phylogenies were used to model habitat preference evolution across two billion years of eukaryotic evolution. Results showed that marine and non-marine organisms are phylogenetically distinct in general, indicating that the salt-barrier is difficult to cross even for microbes. However, microbial eukaryotes have achieved colonizations across marine/non-marine ecosystems several hundred times during their diversification—key events which likely allowed them to colonize vacant ecological niches. Moreover, the results showed that several clades were more adept at transitioning to new habitats, including diatoms, golden algae, and especially fungi. Finally, the researchers could infer the ancestral habitats inhabited by early eukaryotes, and found that the two largest groups, Amorphea and TSAR, likely arose in different habitats. The TSAR lineage (which contains diatoms, ciliates, dinoflagellates, radiolarians etc.) likely arose in the Precambrian oceans, while the ancestral Amorphean (which has now diversified into fungi, animals, choanoflagellates, and amoebozoans) likely inhabited non-marine habitats.

Reviewer: ```suggestion
These large, comprehensive phylogenies were used to model habitat preference evolution across two billion years of eukaryotic evolution. Results showed that marine and non-marine organisms are phylogenetically distinct in general, indicating that the salt-barrier is difficult to cross even for microbes. However, microbial eukaryotes have achieved colonisations across marine/non-marine ecosystems several hundred times during their diversification. These were key events that likely allowed them to colonise vacant ecological niches. Moreover, the results showed that several clades were more adept at transitioning to new habitats, including diatoms, golden algae, and especially fungi. Finally, the researchers could infer the ancestral habitats inhabited by early eukaryotes, and found that the two largest groups, Amorphea and TSAR, likely arose in different habitats. The TSAR lineage (which contains diatoms, ciliates, dinoflagellates, radiolarians etc.) likely arose in the Precambrian oceans, while the ancestral Amorphean (which has now diversified into fungi, animals, choanoflagellates, and amoebozoans) likely inhabited non-marine habitats.
```
